updateProject.html

```diff
@@ -0,0 +1,164 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <base target="_top">
+  </head>
+  <body>
+
+  <h2>Coderbunker Timesheet</h2>
+
+  <br>
+  <style>
+input[type=text] {
+    padding:5px; 
+    border:2px solid #ccc; 
+    -webkit-border-radius: 5px;
+    border-radius: 5px;
+}
+
+input[type=text]:focus {
+    border-color:#333;
+}
+
+input[type=submit] {
+    padding:5px 15px; 
+    background:#ccc; 
+    border:0 none;
+    cursor:pointer;
+    -webkit-border-radius: 5px;
+    border-radius: 5px; 
+}
+
+h2 {
+font-family:Titillium Web;
+}
+label {
+font-family:Titillium Web;
+}
+img {
+width:100px; 
+}
+</style>
+
+   <form onsubmit = "sendAll (event);">
+    <label for = "full name">Enter your full name:</label>
+    <div class="dropdown">
+    <select id="" name = "full name" onfocus = 'this.size=20;' onblur = 'this.size=1;' onchange = 'this.size = 3; this.blur();'>
+    <option value="">Full Name:</option>
+    <option value="Allen Guo">Allen Guo</option>
```

Comment:
hmm, so you don't get the dropdown values from the team sheet?

is there an issue separate for this

Reply:
I just created an issue for this: issue #89